functions/xawslbcontroller/main.k

```diff
@@ -0,0 +1,126 @@
+import models.io.upbound.platform.aws.v1alpha1 as awsv1alpha1
+import models.io.crossplane.helm.v1beta1 as helmv1beta1
+import models.k8s.apimachinery.pkg.apis.meta.v1 as metav1
+import json
+import iam_policy
+
+
+oxr = option("params").oxr # observed composite resource
+_ocds = option("params").ocds # observed composed resources
+_dxr = option("params").dxr # desired composite resource
```

Comment:
do we need `_dxr` and `dcds`?

Reply:
maybe some day. I can remove it, but it comes from the standard template and can be used in case potential extension